static/js/components/Toast.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+export default class Toast extends React.Component {
```

Comment:
is this just a component to pop up a notification to the user? whatever it is, it could definitely use a more descriptive name